### PR TITLE
Add support for custom prefix account keys

### PR DESCRIPTION
**Pull request by anilcse**

### CODE REVIEW

File: relayer/chain.go [hunk 1]
```diff
@@ -286,6 +286,10 @@ func (src *Chain) GetAddress() (sdk.AccAddress, error) {
 		return src.address, nil
 	}
 
+	// Set sdk config to use custom Bech32 account prefix
+	sdkConf := sdk.GetConfig()
```
Comment (jackzampolin) [inline relayer/chain.go:290]: there is a new way to do this. Can you replace these lines with:

```
defer src.UseSDKContext()()
```

Comment (anilCSE) [reply relayer/chain.go:290]: For some reason, this is not working as expected. Debugging . . .

Comment (anilcse) [reply relayer/chain.go:290]: For some reason, this is not working as expected. Debugging . . .

**Comment by jackzampolin** [comment]
Pretty sure this solution solves this @anilCSE https://github.com/iqlusioninc/relayer/pull/179

I've tested and should be working.

**Pull request closed without merging**